bot/cogs/reminders.py

```diff
@@ -165,14 +168,18 @@ def __init__(self, bot: Bot):
             json={
                 'author': ctx.author.id,
                 'channel_id': ctx.message.channel.id,
+                'jump_url': ctx.message.jump_url,
                 'content': content,
                 'expiration': expiration.isoformat()
             }
         )
 
+        now = datetime.utcnow()
+
         # Confirm to the user that it worked.
         await self._send_confirmation(
-            ctx, on_success="Your reminder has been created successfully!"
+            ctx,
+            on_success=f"Your reminder `{content}` will arrive in {humanize_delta(relativedelta(expiration, now))}!"
```

Comment:
I know that adding the contents of the reminder to the confirmation message was mentioned in the issue, but I'm not sure that I like it. We've actively started to remove these type of "confirmation echoes" because the message that sets the reminder will have the exact same contents as what will be echoed here.

I think the only necessary addition is the indication of the date.